hbase-server/src/test/java/org/apache/hadoop/hbase/master/normalizer/TestSimpleRegionNormalizerOnCluster.java

```diff
@@ -179,6 +182,71 @@ void testRegionNormalizationSplitOnCluster(boolean limitedByQuota) throws Except
     admin.deleteTable(TABLENAME);
   }
 
+  // This test is to make sure that normalizer is only going to merge adjacent regions.
+  @Test
+  @SuppressWarnings("deprecation")
```

Comment:
What deprecations are needed here?

Reply:
There is no better way to copy families/values from an existing table descriptor, I tried to use a non-deprecated method, but ModifyableTableDescriptor(tableName, admin.getDescriptor(tableName)) is the best one, which is deprecated. It is important to keep all the parameters to keep previous written data.

Reply:
There's a new API you can use instead of this deprecated one. I think you want something like

```java
    final TableDescriptor td = TableDescriptorBuilder
      .newBuilder(admin.getDescriptor(TABLENAME))
      .setNormalizationEnabled(true)
      .build();
    admin.modifyTable(td);
```

Reply:
Ha, thank you, that is what I was looking for.